README.md

```diff
@@ -289,8 +289,41 @@ All requests over non-SSL connections will be met with a 308 Permanent Redirect.
 HTTP Basic is required for all requests. The password will be ignored, and the
 username should be a unique per client uuid.
 
-The list of allowed clients is part of the configuration, and is passed as a
-comma-separated list to the `ROUTEMASTER_CLIENTS` environment variable.
+All allowed clients are stored in redis. These clients can be listed, added
+and deleted using the `ROUTEMASTER_ROOT_KEY` environment variable. All
+authentication is done with basic auth
+
```

Comment:
I'll also update the "installing and configuring" section which still references the legacy`ROUTEMASTER_CLIENTS`.